truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/impl/Accessor.java

```diff
@@ -420,6 +422,29 @@ protected String toString(TruffleLanguage<?> language, Env env, Object obj) {
         return API.toString(language, env, obj);
     }
 
+    protected boolean supportsOnLoopCount() {
+        return false;
+    }
+
+    protected void onLoopCount(Node source, int count) {
+        // optimized calltarget is not existent on default runtimes
+        if (OPTIMIZEDCALLTARGET != null) {
```

Comment:
We did not need the Accesor in the default runtime yet. But generally this is a good idea. Can we defer this into another changeset?